common/src/main/java/com/wynntils/features/DiscordRichPresenceFeature.java

```diff
@@ -88,7 +89,8 @@ public void onDisconnect(ConnectionEvent.DisconnectedEvent e) {
     }
 
     private void displayCharacterDetails() {
-        int level = Models.CombatXp.getCombatLevel().current();
+        CappedValue combatLevel = Models.CombatXp.getCombatLevel();
+        int level = combatLevel == null ? 0 : combatLevel.current();
```

Comment:
Why did you think this was necessary? getCombatLevel cannot return null.

(In fact, I think we should verify so models *never* return null values)

Reply:
I know something related to this returned null before, as it called player.experienceLevel or something. I will check again in the morning